### PR TITLE
Issue #15218: formatted input file for section 4.8.5.3 methods and constructors annotations to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java [hunk 1]
```diff
@@ -1,90 +1,72 @@
...
+  // ********testing methods.***********
+
+  /** testing. */
+  @SomeAnnotation1
+  @SomeAnnotation2
+  void test1() {}
+
+  /** testing. */
+  @SomeAnnotation1
+  void test2() {}
```
[the remark this answers is not in the document]
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java:17]: @romani @rdiachenko

[the remark this answers is not in the document]
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java:17]: formatter is not opinionated, whatever it does, it does it.
if both code is valid by style guide, we should not raise violations.

as this chapter is about location of annotation, we should revert all changes about annotation location, all other formatting should stay.

**Review by romani — CHANGES_REQUESTED** [review]
only annoation location should be restored.
all other is good formatting - keep it.

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java]: done

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**